completions/bash/kpod

```diff
@@ -273,6 +916,277 @@ _kpod_rename() {
     esac
 }
 
+_kpod_container_run() {
+	local options_with_args="
+		--add-host
+		--attach -a
+		--blkio-weight
+		--blkio-weight-device
+		--cap-add
+		--cap-drop
+		--cgroup-parent
+		--cidfile
+		--cpu-period
+		--cpu-quota
+		--cpu-rt-period
+		--cpu-rt-runtime
+		--cpuset-cpus
+		--cpus
+		--cpuset-mems
+		--cpu-shares -c
+		--device
+		--device-read-bps
+		--device-read-iops
+		--device-write-bps
+		--device-write-iops
+		--dns
+		--dns-option
+		--dns-search
+		--entrypoint
+		--env -e
+		--env-file
+		--expose
+		--group-add
+		--hostname -h
+		--init-path
+		--ip
+		--ip6
+		--ipc
+		--kernel-memory
+		--label-file
+		--label -l
+		--link-local-ip
+		--log-driver
+		--log-opt
+		--mac-address
+		--memory -m
+		--memory-swap
+		--memory-swappiness
+		--memory-reservation
+		--name
+		--network
```

Comment:
we've a "net" option that I think should be network-namespace.  Whatever it becomes, it should be here.